backend/Dockerfile_local

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+#
```

Comment:
the filenames are inconsistent in some cases are with `_` in some others with `-`